bin/collate_qc_data.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+import pandas as pd
+
+from collections import defaultdict
+
+
+def write_summary_qc_report(summary_qc, output_prefix):
+
+    with open(f'{output_prefix}_summary.tab', 'w') as out:
+        out.write('lane_id\tstatus\n')
+        for lane_id, status in summary_qc.items():
+            out.write(f'{lane_id}\t{status}\n')
+
+
+def get_summary_qc(all_reports):
+
+    summary_qc = defaultdict(lambda: 'PASS')
+    for report in all_reports:
+        with open(report, 'r') as file:
+            next(file)
+            for line in file:
+                lane_id = line.split('\t')[0]
+                status = line.split('\n')[0].split('\t')[2]
```

Comment:
Could you re-check that `.split('\t')[2]` should be `2` and not `1`? Ie is it correct that status is the third column?

Reply:
Never mind: I got it that the index is correct.